core/org.eclipse.cdt.ui/plugin.xml

```diff
@@ -4203,6 +4203,12 @@
           description="%HideNonCElements.description"
           id="org.eclipse.cdt.ui.navigator.filters.NonCElementFilter"
           name="%HideNonCElements.label"/>
+    <commonFilter
+          activeByDefault="true"
```

Comment:
@jonahgraham how shall we proceed on a final decision regarding default activation state of this filter?

Reply:
If no one objects, I think active by default as the change makes it visible and it is easy enough to restore old behavior. It is hard for users to discover new features, and if we don't activate by default most people won't find it.

If someone objects, then we can have a formal vote on cdt-dev.